nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'There is no implemented method'
+  end
```

Comment:
- [ ] It would be nice when you raise an error that the user can see what class and which method is giving the error, you can do this by using string interpolation as shown in the below example.
```
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end

```

Reply:
Thank you.